manim/mobject/geometry.py

```diff
@@ -2206,6 +2206,10 @@ class Rectangle(Polygon):
         The vertical height of the rectangle.
     width : :class:`float`, optional
         The horizontal width of the rectangle.
+    grid_xstep : :class:`float`, optional
```

Comment:
Add typings in the function parameters
See: https://docs.manim.community/en/latest/contributing/typings.html

Reply:
Like for `height` and `width` ;-) ? No offense, just saying... I will add them